modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -489,7 +528,7 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::checkInputWindow(const GenericReal<is_ad>
     switch (behavior)
     {
       case WindowFailure::WARN:
-        mooseWarning(msg.str());
+        mooseDoOnce(mooseWarning(msg.str()));
```

Comment:
@tophmatthews are you okay with this change? We were thinking the warning output can become too invasive when we evaluate the model outside the range and that, hopefully, printing it out once would be enough to alert the user. Let me know if you think otherwise and I'll revert it.

Reply:
Isn't mooseWarning already wrapped in a DoOnce?

Reply:
`mooseDeprecatedStream` and `mooseInfoStream` are, but not `mooseWarning`.

Reply:
Yeah, this is probably fine. We're working hard on removing the need for this, so seems good to me.